### PR TITLE
Fix small regression bugs and "the week of"

### PR DESCRIPTION
**Pull request by rubio41**
- Add 649 midnight tonight. 
- ADD LUNCH AT 12:30 PM ON FRI 
- For four Monday at noon
- What do I have the week of November 30th

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.3
+# Visual Studio 14
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.sln]: Let's not change this back to older versions.

File: .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs [hunk 2]
```diff
@@ -214,12 +215,14 @@ public void TestTimeParseLuis()
         [TestMethod]
         public void TestTimeParseMealTimeLuis()
         {
-            BasicTest("I'll be back 12 o'clock lunchtime", "T12");
-            BasicTest("I'll be back 12 o'clock lunch", "T12");
-            BasicTest("I'll be back 8 o'clock dinner", "T20");
-            BasicTest("I'll be back 8 o'clock breakfast", "T08");
-            BasicTest("I'll be back at lunch 12 o'clock", "T12");
+            //We only support lunchtime now
+            //BasicTest("I'll be back 12 o'clock lunch", "T12");
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs]: We'll add a task to potentially bring some of these back only as disambiguation.

File: .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestDatePeriodExtractor.cs [hunk 3]
```diff
@@ -181,6 +181,7 @@ public void TestDatePeriodExtractSeason()
         public void TestDatePeriodExtractWeekOf()
         {
             //test week of and month of
+            BasicTest("the week of september.15th", 0, 26);
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestDatePeriodExtractor.cs]: In new unit tests we should add longer more complete sentences. That will help make sure corner cases don't happen.

File: Specs/DateTime/English/DateTimeModel.json [hunk 4]
```diff
@@ -415,7 +415,6 @@
   "Context": {
     "ReferenceDateTime": "2016-11-07T00:00:00"
   },
-  "NotSupported": "javascript",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json]: Can you revert these removal changes? These tests should still be marked as not supported in JS.

File: Specs/NumberWithUnit/English/DimensionModel.json [hunk 5]
```diff
@@ -570,7 +570,6 @@
 ,
 {
   "Input": "in 1995 canon introduced the first commercially available slr lens with internal image stabilization , ef 75 -300mm f / 4 - 5 . 6 is usm.",
-  "NotSupported": "javascript",
```
Comment (tellarin) [inline Specs/NumberWithUnit/English/DimensionModel.json]: Same.

File: .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs (hunk 2 above)
Comment (rubio41) [reply .NET/Microsoft.Recognizers.Text.DateTime.Tests/English/TestTimeParser.cs]: done
